src/agent/Disk.h

```diff
@@ -6,16 +6,10 @@ class Disk
 {
 public:
     Disk() = default;
-    Disk(const std::string& _caption, const std::string& _deviceId, const std::string& _model, const int& _partitions, const long long& _size);
+    Disk(const std::string& _deviceId);
 
-    const std::string& caption() const;
     const std::string& deviceId() const;
-    const std::string& model() const;
 
 protected:

```

Comment:
somethin I have missed recently: private

Reply:
With private DiskMock will not work.